gm4_damaging_stonecutters/data/damaging_stonecutters/functions/damage.mcfunction

```diff
@@ -0,0 +1,11 @@
+#damages entity
+#@s - @e[type=!#damaging_stonecutters:no_target]
+#called by damaging_stonecutters:check_damage
+
+#damage entity
+effect give @s minecraft:absorption 1 0 true
+effect give @s[type=!#damaging_stonecutters:undead] minecraft:instant_damage 1 0 true
+effect give @s[type=#damaging_stonecutters:undead] minecraft:instant_health 1 0 true
+
+#audiovisuals
+function damaging_stonecutters:audiovisuals
```

Comment:
Instead of using a different function, put the commands in here directly.

Reply:
That's a remnant from when the audiovisuals were used in more than one function